docs/use-cases/machine-learning.md

```diff
@@ -0,0 +1,13 @@
+# Machine Learning
+
+## Docs
+
+* [Quick start](../quick-start.md) and [training](../training.md)
+* Try out the updated [Python and Java SDKs](https://github.com/argoproj/argo-workflows/tree/master/sdks).
+* [Authoring and Submitting Argo Workflows using Python](https://blog.argoproj.io/authoring-and-submitting-argo-workflows-using-python-aff9a070d95f)
+* Head to the [Kubeflow docs](https://www.kubeflow.org/docs/components/pipelines/).
+
+## Videos
```

Comment:
There are several new ones for ML in https://github.com/terrytangyuan/awesome-argo#videos. Search for "Machine Learning" there.